src/ddl/v4_schema.sql

```diff
@@ -0,0 +1,212 @@
+-- --------------------------------
+-- TODO: REMOVE THESE HACKS!!!  (find a better way to do this
+-- 
+-- the database schema `epidata` is created by ENV variables specified in the docker image definition found at:
+--     ../../dev/docker/database/epidata/Dockerfile
+-- and the user 'user' is created with permissions on that database.
+-- here we create the `covid` schema and extend permissions to the same user,
+-- as the ENV options do not appear to be expressive enough to do this as well.
+-- this is incredibly permissive and easily guessable, but is reqd for testing our environment.
+--
+CREATE DATABASE covid;
+USE covid;
+GRANT ALL ON covid.* TO 'user';
+-- END TODO
+-- --------------------------------
+
```

Comment:
yes, removing all of this will cause CI to fail.  i considered (and have not yet eliminated as a possibility) putting the CREATE and GRANT lines into another .sql file in this directory, but the USE will still be required (or another trick, like specifying the schema along with each table name (like `covid.signal_history`)).